crates/eko/src/lib.rs

```diff
@@ -205,6 +238,7 @@ pub unsafe extern "C" fn rust_quad_ker(u: f64, rargs: *mut c_void) -> f64 {
         args.a_half_x,
         args.a_half_y,
         args.alphaem_running,
+        args.n3lo_ad_variation,
```

Comment:
There is no need to pass the `n3lo_ad_variation` back to python - they are consumed here

Reply:
I should have fixed.